apsuite/commisslib/meas_filling_pattern.py

```diff
@@ -0,0 +1,130 @@
+"""."""
+
+import socket as _socket
+import time as _time
+import sys as _sys
+import numpy as _np
+
+
+from mathphys.functions import get_namedtuple as _get_namedtuple
+
+
+class MeasFillingPattern:
```

Comment:
Doesn't this class looks more like something to go in our `siriuspy.devices` package?

Reply:
closing this PR in favor of https://github.com/lnls-sirius/dev-packages/pull/778